report.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'board'
+require_relative 'burndown'
+require_relative 'helpers'
+
+module Spregen
+  class Report
+
+    include Spregen::Helpers
+
+    attr_accessor :report
+
+    def initialize(configuration = {})
+      Spregen::Config.configure_trello_client
```

Comment:
This class have to be as generic as possible. Thus, I think any client reference (trello configuration in this case) must be on the Board class.